src/components/LoanCards/LoanTags/LoanCallouts.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+	<div class="tw-overflow-hidden" style="width: inherit;">
```

Comment:
Long names truncate in the pill, which is good, but If there are two long pill names, the second will no longer be a pill. Here's an extreme version. This could be a follow-up ticket.

![image](https://user-images.githubusercontent.com/16867161/217934820-375ba1f5-2a10-4a76-9fe1-2acda3978fe8.png)

Reply:
Callouts are now 50% width max w/ ellipsis